apollo-router/Cargo.toml

```diff
@@ -136,11 +136,11 @@ tower-http = { version = "0.3.4", features = [
 ] }
 tower-service = "0.3.2"
 tower-test = "0.4.0"
-tracing = "=0.1.34"
-tracing-core = "=0.1.26"
+tracing = "0.1.34"
+tracing-core = "0.1.29"
```

Comment:
we pinned those dependencies through [this PR](https://github.com/apollographql/router/pull/1410) for [that reason](https://github.com/apollographql/router/issues/1407).

Do you know for sure this won't be an issue anymore and that updating is safe?

Reply:
right, I'm reverting that in a955ba9a. It looks like the CI passes, but that's something we can investigate separately